libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -25,77 +25,50 @@
 #include <cstdint>
 #include <limits>
 
+static const int64_t N{10000};
+
 using namespace fetch::fixed_point;
 
+using MyFPTypes = ::testing::Types<fetch::fixed_point::fp32_t, fetch::fixed_point::fp64_t,
```

Comment:
please rename this to FloatingPointTypes and move this to math/tests/unit/test_types.hpp

Reply:
I explicitly named them as FixedPointTypes instead and moved to the file as you suggested.